app/assets/stylesheets/boards.scss

```diff
@@ -0,0 +1,27 @@
+ 	.board {
+ 		width: 60%;
+    	height: 70%;
+    	border: 3px solid #333;
+    	margin: 10px auto 10px auto;
+
```

Comment:
I think he primarily wanted us to avoid using Bootstrap columns/rows for the actual board, but it should be fine to wrap all the content in a Bootstrap container. In any event, I would still apply the border and margins directly to the table rather than a wrapper div.